frb/tests/test_galfit.py

```diff
@@ -0,0 +1,45 @@
+# Module to test 
+# the galfit wrapper
+
+import pytest
+import os
+import shutil
+import numpy as np
+
+from astropy.io import fits
+from astropy.table import Table
+from astropy.wcs import WCS
+
+from pkg_resources import resource_filename
+
+from frb.galaxies.frbgalaxy import FRBHost
+
+try:
+    from frb.galaxies import galfit as glf
+except ImportError:
```

Comment:
If this test is going to crash without this import, then best to have
it crash out here.

Reply:
Done.